rules/windows/process_creation/win_susp_process_creations.yml

```diff
@@ -24,6 +25,17 @@ logsource:
 detection:
     selection:
         CommandLine:
+            - vssadmin.exe delete shadows*
+            - vssadmin delete shadows*
```

Comment:
duplicate with:

- [rules/windows/process_creation/win_ransomware_shadowcopy.yml](https://github.com/Neo23x0/sigma/blob/oscd/rules/windows/process_creation/win_ransomware_shadowcopy.yml)
- [rules/windows/process_creation/win_shadow_copies_deletion.yml](https://github.com/Neo23x0/sigma/blob/oscd/rules/windows/process_creation/win_shadow_copies_deletion.yml)

I think better to left the [rules/windows/process_creation/win_shadow_copies_deletion.yml](https://github.com/Neo23x0/sigma/blob/oscd/rules/windows/process_creation/win_shadow_copies_deletion.yml) rule since it already includes all authors, better constructed and wider than [rules/windows/process_creation/win_ransomware_shadowcopy.yml](https://github.com/Neo23x0/sigma/blob/oscd/rules/windows/process_creation/win_ransomware_shadowcopy.yml)